### DISCUSSION TITLE
Run relagit with hot-reload do not update app when realoding it ( ctrl +r)

### DISCUSSION THREAD

**Discussion by WeeskyBDW** [relagit/relagit#13] [Q&A] [answered]
Hello
Nota: i never used electron in a project so it's my first time dealing with it. however i do know Javascript (both client-side and server/node-side) and React
sorry idk if this place is the best to ask that but i have a problem when running relagit in development mode (for understanding app code why not helping with development when i'll have more time):

When i do `pnpm run dev` it didn't open the app. When i try to open manually public/index.html, it work but dont take change (even if i reload app or restart it). Am i doing something wrong ?

**Comment by TheCommieAxolotl** [comment, marked as answer]
`pnpm dev` doesn't open the app, you must also run `pnpm start`

**Reply by TheCommieAxolotl** [reply]
If you're working in vscode, you can also use the "Run and Debug" tool to run the app.
